### ISSUE TITLE
Umbraco Forms 8.7.6 - Can nolonger overwrite on a form by form basis

### ISSUE THREAD

**Issue by Ric-Carey** [umbraco/Umbraco-CMS#10783]
### Which *exact* Umbraco version are you using? For example: 8.13.1 - don't just write v8

8.14.2

### Bug summary

Hello,

In previous versions of Umbraco Forms, you used to be able to target a specific form and overwrite the views specific to that form by using a GUID in the folder structure. E.g docs https://our.umbraco.com/documentation/Add-ons/umbracoforms/developer/custom-markup/#customizing-for-a-specific-form 

It appears you no longer can do this. We have rendered the form using:

@Umbraco.RenderMacro("renderUmbracoForm", new { FormGuid = Model.UmbracoForm.ToString(), FormTheme = "PROJECT", ExcludeScripts = "1" })

Then we would like to override the form.cshtml file from our theme for a specific form, however this doesn’t appear to work. As the original theme form.cshtml is present.

We have tried the usual ~\Views\Partials\Forms\GUID\Form.cshtml as well as ~\Views\Partials\Forms\Themes\PROJECT\GUID\Form.cshtmland neither work.

Many thanks,
Ric

### Steps to reproduce

As above.

**Comment by bjarnef** [comment]
I don't think you should have the Guid folder. At least I haven't used that approach (not sure if that has worked previous though).

Instead just use include a theme folder name, e.g.

`~/Views/Partials/Forms\
/Themes/MyTheme/Form.cshtml`

And use the specific theme:
`@Umbraco.RenderMacro("renderUmbracoForm", new { FormGuid = Model.UmbracoForm.ToString(), FormTheme = "MyTheme", ExcludeScripts = "1" })`
